docker/build/requirements.txt

```diff
@@ -5,9 +5,9 @@ boto==2.48.0  # Tile generation
 bottle==0.12.13  # i18n
 c2c.cssmin==0.7.dev6  # CGXP build
 c2c.template==1.5.1  # geoportal
-https://github.com/camptocamp/c2cgeoform/archive/87a4191ef3330b76497bd009e9c0220cbc73c625.zip#egg=c2cgeoform  # commons
+-e git://github.com/camptocamp/c2cgeoform.git@b6d4bbc78da55bf270ae62ee3eaac77d5dc92062#egg=c2cgeoform  # commons
```

Comment:
Don't use the git protocol, it's working only it we have an ssh key, ant he archive way download less data ...

Reply:
But if we use the git protocol, this will work behind proxies (whereas https: will create problems), IIUC?

Reply:
@jwkaltz: I do not understand your comments, proxy usually allow https but not git, in aeroportdelyon_sig we are behin a proxy and we use: 
```
git config --global url."https://".insteadOf git://
```
So this is transparent except you have to give your credentials for private repositories.

Reply:
@arnaud-morvan I think I mixed things up. I was thinking about https://github.com/camptocamp/ngeo/pull/2876